breathe/directives.py

```diff
@@ -319,7 +319,8 @@ class _DoxygenContentBlockDirective(BaseDirective):
         "protected-members": flag,
         "private-members": flag,
         "undoc-members": flag,
-        "no-link": flag
+        "no-link": flag,
+        "description-only": flag
```

Comment:
@intelkevinputnam On a side note, I'm also fine (and perhaps prefer?) `desc-only`, helps keep the flag length a bit more reasonable just like with `undoc-members`.

Reply:
Sounds good. I'll add docs, rebase, and resubmit.